packages/vuetify/src/components/VSwitch/VSwitch.tsx

```diff
@@ -158,7 +159,7 @@ export const VSwitch = genericComponent<new <T>(
                       <div
                         class={[
                           'v-switch__track',
-                          ...backgroundColorClasses.value,
+                          ...(!isForcedColorsModeActive ? backgroundColorClasses.value : ''),
```

Comment:
```suggestion
                          !isForcedColorsModeActive ? backgroundColorClasses.value : undefined,
```

Reply:
I'm not sure why backgroundColorClasses is being spread here in the first place. You can adjust the logic as suggested.

Reply:
Done